src/main/scala/mesosphere/marathon/AcceptedResourceRolesDefaultBehavior.scala

```diff
@@ -0,0 +1,39 @@
+package mesosphere.marathon
+
+/**
+  * The accepted resource roles default behavior defines whether which values are selected for the accepted resource roles by default
+  *
+  *  - [[AcceptedResourceRolesDefaultBehavior.Any]] indicates that either reserved or unreserved resources will be accepted. (default)
+  *  - [[AcceptedResourceRolesDefaultBehavior.Unreserved]] Only unreserved ( offers with role '*' ) will be accepted.
+  *  - [[AcceptedResourceRolesDefaultBehavior.Reserved]] Only reserved (offers with the role of the starting task) will be accepted.
+  *
+  */
+sealed trait AcceptedResourceRolesDefaultBehavior {
```

Comment:
this is odd naming IMO... this is a "configured" "Default behavior" which will lead to a discussion of what is the default default behavior vs the configured default behavior... which if changed isn't the default :)

Reply:
can we call this `AcceptedResourceRolesBehavior`?

Reply:
It's the behavior of the defaulting mechanism, not the behavior of accepted resource roles, once specified. I know it's a weird name. I'm not sure removing the Default makes it clearer